test/components/avatar_component_test.rb

```diff
@@ -46,7 +46,7 @@ def test_squared_avatar
     render_inline(Primer::AvatarComponent.new(src: "https://github.com/github.png", alt: "github", square: true))
 
     assert_selector("img.avatar")
-    refute_selector(".CircleBadge")
+    refute_selector(".circle")
```

Comment:
It appeared that this should have been changed when AvatarComponent moved from using the CircleBadge class to circle.